src/plugins/expression/expression-buttons.scss

```diff
@@ -0,0 +1,14 @@
+@import "../../components/vars.sass";
+
+.expression-button {
+  display: flex;
+  &:hover {
+    background-color: $workspace-teal-light-5 !important;
```

Comment:
Do we need `!important` here? 😬

Reply:
Thanks for catching that.  It was not needed and was an not-cleaned-up artifact of debugging